### ISSUE TITLE
Ignore Option Not Working on Windows

### ISSUE THREAD

**Issue by evanlknapp** [nilslice/protolock#94]
I was using protolock locally when I started having issues with ignoring directories.  I found eventually that the parse_test.go tests were failing for me on windows (with no local changes) and that stepping through the code things weren't behaving right.

I think I limited it down to this [line](https://github.com/nilslice/protolock/blob/ddda3438d08cd2a15c32f36c1b5c305bd08fbcc1/parse.go#L498) not working correctly.
```
if !strings.HasPrefix(rel, "../") {
```

My assumption is that this line is making a big assumption about path separators depending on what the operating system in use is.  When I changed it to be this:
```
if !strings.HasPrefix(rel, ".." + string(os.PathSeparator)) {
```

Everything appeared to work as expected.

**Comment by nilslice** [comment]
Ah, great catch. Blame it on my poor PR reviews :) 

Would you like to submit a patch PR? Thank you for finding this issue!

**Comment by evanlknapp** [comment]
Yeah no problem. I am not going to add any tests as the existing tests should be sufficient.

Here's my PR: #95

**Comment by nilslice** [comment]
merged in #97!

**Comment by evanlknapp** [comment]
@nilslice awesome! Thanks!
